src/Components/CardboardMultiSelect/CardboardMultiSelect.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useCallback, useState } from 'react';
+import {
+    ICardboardMultiSelectProps,
+    ICardboardMultiSelectStyleProps,
+    ICardboardMultiSelectStyles,
+    MultiselectOption
+} from './CardboardMultiSelect.types';
+import { getMultiSelectStyles, getStyles } from './CardboardMultiSelect.styles';
+import { classNamesFunction, useTheme, styled } from '@fluentui/react';
+import CreatableSelect from 'react-select/creatable';
+import { OnChangeValue } from 'react-select';
+import { useTranslation } from 'react-i18next';
+
+const getClassNames = classNamesFunction<
+    ICardboardMultiSelectStyleProps,
+    ICardboardMultiSelectStyles
+>();
+
+const components = {
+    DropdownIndicator: null
+};
+
+const createOption = (label: string): MultiselectOption => ({
+    label,
+    value: label
+});
+
+const ROOT_LOC = '3dSceneBuilder.visualRuleForm';
+const LOC_KEYS = {
+    placeholderMultiselect: `${ROOT_LOC}.placeholderMultiselect`
+};
+
+const CardboardMultiSelect: React.FC<ICardboardMultiSelectProps> = (props) => {
+    const { currentValues, onChangeValues, styles } = props;
+
+    // contexts
+
+    // state
+    const [inputValue, setInputValue] = useState('');
+    const [selectedValues, setSelectedValues] = useState<MultiselectOption[]>(
+        currentValues
+            ? (currentValues as string[]).map((value) => createOption(value))
+            : []
+    );
+
+    // hooks
+    const { t } = useTranslation();
+
+    // callbacks
+    const handleKeyDown = useCallback(
+        (event: React.KeyboardEvent<HTMLDivElement>) => {
+            if (!inputValue) return;
+            switch (event.key) {
+                case 'Enter':
+                case 'Tab': {
```

Comment:
tab too? That doesn't feel like standard behavior to me. Can you still tab out of the control?

Reply:
good catch!